### PR TITLE
Device Monitoring Documentation

### PR DESCRIPTION
**Pull request by dashpole**
Placeholder for documentation for https://github.com/kubernetes/features/issues/606.  Documentation will be added after the proposal is merged.

### CODE REVIEW

**Comment by zparnold** [comment]
I expect this to merge this week. 😄 @xiangpengzhao @dashpole When you have a moment this week, can I have you fill this in with docs on Device Monitoring?

**Comment by dashpole** [comment]
The Device Monitoring feature has been delayed until 1.13

**Comment by tfogo** [comment]
Hi @dashpole, could you please complete this ASAP and let us know when it's ready for review? The docs deadline is approaching soon and we want to start reviewing PRs ASAP. Thanks!

**Comment by dashpole** [comment]
@tfogo ack.  I will do this today

File: content/en/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins.md [hunk 1]
```diff
@@ -136,6 +136,36 @@ a Kubernetes release with a newer device plugin API version, upgrade your device
...
 
+## Monitoring Device Plugin Resources
+
+In order to monitor resources provided by device plugins, monitoring agents need to be able to 
+discover the set of devices that are in-use on the node and obtain metadata to describe which 
+container the metric should be associated with.  Prometheus metrics exposed by device monitoring 
+agents should follow the 
+[Kubernetes Instrumentation Guidelines](https://github.com/kubernetes/community/blob/master/contributors/devel/instrumentation.md), 
+which requires identifying containers using `pod`, `namespace`, and `container` prometheus labels.  
+The kubelet provides a grpc service to enable discovery of in-use devices, and to provide metadata
```
Comment (tengqm) [inline content/en/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins.md]: grpc -> gRPC

File: content/en/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins.md [hunk 2]
```diff
@@ -136,6 +136,36 @@ a Kubernetes release with a newer device plugin API version, upgrade your device
...
+The gRPC service is served over a unix socket at `/var/lib/kubelet/pod-resources/kubelet.sock`. 
+Monitoring agents for device plugin resources can be deployed as a daemon, or as a DaemonSet. 
+The cannonical directory `/var/lib/kubelet/pod-resources` requires privileged access, so monitoring 
+agents must run in a privileged security context.  If a device monitoring agent is running as a 
+DaemonSet, `/var/lib/kubelet/pod-resources` must be mounted as a 
+[Volume](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#volume-v1-core)
+in the plugin's
+[PodSpec](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#podspec-v1-core).
+
+Support for the pod-resources service is still in alpha.
```
Comment (tengqm) [inline content/en/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins.md]: "pod-resources service"?

**Pull request merged**